### ISSUE TITLE
limjh16/jekyll-action-ts@v2 fails due to missing /usr/local/bin/bundle

### ISSUE THREAD

**Issue by anthonyfok** [OpenDRR/documentation#13]
Recent attempt to run our jekyll-with-asciidoc.yml GitHub workflow to build the OpenDRR/documentation site fails due to limjh16/jekyll-action-ts@v2 not being able to find the executable `bundle`.

Apparently, in recent updates to GitHub's ubuntu-20.04 virtual environment, Ruby Bundler /usr/local/bin/bundle (and perhaps /usr/local/bin/bundler too) have been removed.

The last successful run was on 2021-04-22 at https://github.com/OpenDRR/documentation/actions/runs/774977268:

> #### Virtual Environment
>  Environment: ubuntu-20.04
>  Version: 20210419.1
>  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210419.1/images/linux/Ubuntu2004->README.md
>  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210419.1

The current failed run on 2021-06-10: https://github.com/OpenDRR/documentation/actions/runs/925909471

> #### Virtual Environment
> Virtual Environment
>   Environment: ubuntu-20.04
>   Version: 20210606.1
>   Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210606.1/images/linux/Ubuntu2004-README.md
>   Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210606.1

I wasn't able to find any documentation about the removal of `bundle` in Ubuntu2004-README.md though.

A quick fix may be to `apt-get install -y ruby-bundler`.

**Comment by anthonyfok** [comment]
For reference: Ruby Bundler is still installed _somewhere_, in multiple locations, even as /usr/bin/bundle2.7 and /usr/bin/bundler2.7, but `/usr/local/bin/bundle` and `/usr/local/bin/bundler` are no more:

```
[-rwxrwxrwx runner   docker          2841 Apr  5 11:46]  /opt/hostedtoolcache/Ruby/2.5.9/x64/lib/ruby/gems/2.5.0/gems/rake-12.3.3/bin/bundle
[-rwxrwxrwx runner   docker           557 Apr  5 17:00]  /opt/hostedtoolcache/Ruby/2.6.7/x64/bin/bundle
[-rwxrwxrwx runner   docker           559 Apr  5 17:00]  /opt/hostedtoolcache/Ruby/2.6.7/x64/bin/bundler
[-rwxrwxrwx runner   docker          2841 Apr  5 11:48]  /opt/hostedtoolcache/Ruby/2.6.7/x64/lib/ruby/gems/2.6.0/gems/rake-12.3.3/bin/bundle
[-rwxrwxrwx runner   docker           557 Apr  5 17:00]  /opt/hostedtoolcache/Ruby/2.7.3/x64/bin/bundle
[-rwxrwxrwx runner   docker           559 Apr  5 17:00]  /opt/hostedtoolcache/Ruby/2.7.3/x64/bin/bundler
[-rwxrwxrwx runner   docker          2841 Apr  5 12:39]  /opt/hostedtoolcache/Ruby/2.7.3/x64/lib/ruby/gems/2.7.0/gems/rake-13.0.1/bin/bundle
[-rwxrwxrwx runner   docker           557 Apr  5 17:01]  /opt/hostedtoolcache/Ruby/3.0.1/x64/bin/bundle
[-rwxrwxrwx runner   docker           559 Apr  5 17:01]  /opt/hostedtoolcache/Ruby/3.0.1/x64/bin/bundler
[-rwxrwxrwx runner   docker          2841 Apr  5 12:33]  /opt/hostedtoolcache/Ruby/3.0.1/x64/lib/ruby/gems/3.0.0/gems/rake-13.0.3/bin/bundle
[-rwxr-xr-x root     root             529 Apr 15 14:38]  /usr/bin/bundle2.7
[-rwxr-xr-x root     root             531 Apr 15 14:38]  /usr/bin/bundler2.7
[-rwxrwxrwx root     root            2841 Apr  1  2020]  /usr/share/rubygems-integration/all/gems/rake-13.0.1/bin/bundle
```

**Comment by anthonyfok** [comment]
Hmm... strange... GitHub did not automatically close this issue after I merged PR #14... I wonder why...

Maybe it is a sign for me to use the https://github.com/ruby/setup-ruby action?  :rofl:  I'll keep this issue open for a little longer yet then.

**Comment by anthonyfok** [comment]
Fixed in [commt 4c8a8d5](https://github.com/OpenDRR/documentation/commit/4c8a8d5e040b24523ffecdbafa7de6d13add1c17) in the **gh-pages-source** branch.
